component-lib/.gitignore

```diff
@@ -0,0 +1 @@
+yarn.lock
```

Comment:
`node_modules` is excluded at the project level (the .gitignore in the parent directory), but `yarn.lock` isn't. I don't feel strongly about whether this ignore moves up a level or not. It could be argued that it breaks convention, so I'm happy to move it up.

Reply:
Oh! I'm mistaken. We intentionally don't commit our Pipfile.lock - but we _do_ commit `yarn.lock` for the frontend. So I'll just remove this gitignore!